indico/modules/events/sessions/__init__.py

```diff
@@ -111,3 +112,11 @@ class CoordinatorRole(ManagementRole):
     name = 'coordinate'
     friendly_name = _('Coordination')
     description = _('Grants coordination access to the session.')
+
+
+@signals.event.sidemenu.connect
+def _extend_event_menu(sender, **kwargs):
+    def visible_my_sessions(event):
+        return bool(get_sessions_for_user(event.as_event, session.user))
```

Comment:
`session.user and bool(get_sessions_for_user(event.as_event, session.user))` - no need to do stuff in the function if there's no user